### ISSUE TITLE
[Bug] Flutter Web build fails due to Platform usage in theme

### ISSUE THREAD

**Issue by Tayebsed93** [Orange-OpenSource/ouds-flutter#300]
### Description:
The app fails to compile on Flutter Web. 
Because `dart:io’s` Platform is used in the theme (e.g., Platform.isAndroid), which is unsupported on Web :

Error message : `Unsupported operation: Platform._operatingSystem`

### Steps to reproduce:
Run the app on Web (e.g., flutter run -d chrome).

### Expected behavior:
Use `kIsWeb` and `defaultTargetPlatform` from `package:flutter/foundation.dart` for platform checks instead of Platform.

### Suggested fix:

**Orange theme :** 
```
import 'package:flutter/foundation.dart';

@override
String get fontFamily => defaultTargetPlatform == TargetPlatform.android ? 'Roboto' : 'SFProDisplay';
```

**Comment by B3nz01d** [comment]
Closed during OUDS sprint # 20
